### PR TITLE
App drawer transfer

### PR DESCRIPTION
**Pull request by robdodson**
ok sorry for the dumb branch names but we should be able to LGTM this PR and continue work in the app-drawer-migration branch before we cut over to master

@abdonrd @chuckh

### CODE REVIEW

**Comment by chuckh** [comment]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,390 +1,83 @@
-![](https://cloud.githubusercontent.com/assets/110953/7877439/6a69d03e-0590-11e5-9fac-c614246606de.png)
-## Polymer Starter Kit
+# Polymer App Toolbox - Starter Kit
 
-> A starting point for building web applications with Polymer 1.0
+This template is a starting point for building apps using a drawer-based
+layout.  The layout is provided by `app-layout` elements.
```
Comment (abdonrd) [inline README.md]: Double space before `The`.

File: README.md [hunk 2]
```diff
@@ -1,390 +1,83 @@
...
+    mkdir my-app
+    cd my-app
+    polymer init starter-kit
 
-### Get the code
+### Start the development server
 
-[Download](https://github.com/polymerelements/polymer-starter-kit/releases/latest) and extract Polymer Starter Kit to where you want to work. The project comes in two flavours - Light and Full.
+This command serves the app at `http://localhost:8080` and provides basic URL
+routing for the app:
```
Comment (abdonrd) [inline README.md]: Extra breakline.

File: sw-precache-config.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (abdonrd) [inline sw-precache-config.js]: We need this file?

File: README.md [hunk 4]
```diff
@@ -57,11 +57,13 @@ H2/push-compatible servers or to clients that do not support H2/Push.
 This command serves the minified version of the app in an unbundled state, as it would
 be served by a push-compatible server:
 
-    polymer serve build/unbundled --open
```
Comment (abdonrd) [inline README.md]: Why this doesn't work? 😕

File: .travis.yml [hunk 5]
```diff
@@ -2,27 +2,18 @@ language: node_js
 sudo: required
 dist: trusty
 addons:
-  firefox: latest
+  firefox: 46
```
Comment (chuckh) [inline .travis.yml]: `firefox: 46` should be `firefox: '46.0'`. See https://github.com/PolymerElements/polymer-starter-kit/blob/psk2/.travis.yml#L5.

File: .travis.yml [hunk 6]
```diff
@@ -2,27 +2,18 @@ language: node_js
...
 node_js:
-  - '0.12'
-  - '4.2'
-  - '5.1'
-cache:
-  directories:
-    - node_modules
+  - "6"
+  - "5"
+  - "4"
```
Comment (chuckh) [inline .travis.yml]: Use single quotes for 6,5,4.

File: sw-precache-config.js (hunk 3 above)
Comment (robdodson) [inline sw-precache-config.js]: good question.. @FredKSchott I noticed this was in the app-drawer-template repo, do you know if we still need it or will CLI do the precache config for us?

Comment (robdodson) [inline sw-precache-config.js]: note this branch does not use polymer-build, it's just vanilla CLI

File: README.md (hunk 1 above)
Comment (robdodson) [inline README.md]: Good catch

File: LICENSE.txt [hunk 7]
```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016 The Polymer Authors. All rights reserved.
```
Comment (abdonrd) [inline LICENSE.txt]: Why the `//` ?

File: sw-precache-config.js (hunk 3 above)
Comment (FredKSchott) [inline sw-precache-config.js]: Needed for the cli to build a service worker
Not needed by polymer-build

File: LICENSE.txt (hunk 7 above)
Comment (robdodson) [inline LICENSE.txt]: I don't know, i just copied the LICENSE from https://github.com/Polymer/polymer/blob/master/LICENSE.txt

**Pull request merged**